backend/utils/telemetry.py

```diff
@@ -20,7 +22,11 @@ def send(self):
         if settings.TELEMETRY_ON:
             self.thread = Thread(target=self._post_request)
             self.thread.run()
+        else:
+            log.info(f"Telemetry is switched off. Not sending telemetry data for project {self.data['uuid']}.")
         
     def _post_request(self):
-        r = requests.post(self.url, data=json.dumps(self.data))
+        log.info(f"Sending telemetry data for project {self.data['uuid']} to {self.url}.")
+        r = requests.post(self.url, data=self.data)
```

Comment:
This probably wants to be `json=` rather than `data=` in order to post the telemetry data as `application/json` rather than `x-www-form-urlencoded`